src/INSTRUCTIONS.md

```diff
@@ -29,4 +29,13 @@ The `QuestionOne` component has a `service` prop - This is a data service which
 
 The goal is to see how candidates think through form submissions and what scenarios a user is likely to encounter.
 
-We have included unit tests to validate your solution - Run these from the terminal with `yarn test`. If you think there are additional scenarios to test for then feel free to add some!
+We have included unit tests to validate your solution - Run these from the terminal with `yarn test`. If you think there are additional scenarios to test for then feel free to add some, if you're not used to testing your code have a go at it anyway and feel free to talk it through in the follow up or in person if youre pairing.
+
+
+Spend around an hour on this, don't feel you have to rush a solution or complete everything; you will not be judged on whether you finish or not. We would much rather you give us a solution that you are proud of and feel best displays the type of code you would want to ship into production. That said, make sure there is a way to run your app and it does give some sort of output.
```

Comment:
Not sure the bit about running the app is relevant - All of that is already in place, they would have to do something quite wrong to cause the app not to run at all.

```suggestion
Spend around an hour on this, don't feel you have to rush a solution or complete everything; you will not be judged on whether you finish or not. We would much rather you give us a solution that you are proud of and feel best displays the type of code you would want to ship into production.
```